src/robot/conf/settings.py

```diff
@@ -115,7 +115,12 @@ def _process_value(self, name, value):
             return utils.abspath(value)
         if name in ['SuiteStatLevel', 'MonitorWidth']:
             return self._convert_to_positive_integer_or_default(name, value)
-        if name in ['Listeners', 'VariableFiles']:
+        if name == 'Listeners':
+            if isinstance(value, list):
+                return [self._split_args_from_name_or_path(item) for item in value]
+            else:
+                return [(value, [])]
```

Comment:
Do I get it right that it is possible to give only a single listener object but not a list of them? Fixing that would require changes to how individually given options are turned into a list. Currently there's a check

    if default == [] and isinstance(value, basestring):

on line 77 in this module. Changing that to

    if default == [] and not is_list_line(value):

would probably be enough. This obviously should also be tested.